module/profile/helper.js

```diff
@@ -0,0 +1,134 @@
+// calling entities service for entity-managament
+const entityManagementService = require(GENERICS_FILES_PATH + '/services/entity-management')
+// calling user service api
+const userService = require(GENERICS_FILES_PATH + '/services/users')
+
+module.exports = class FormsHelper {
```

Comment:
@MallanagoudaB Why the class name is FormHelper please change that to ProfileHelper and test the flow